jobs/integr8ly/browser-based-testsuite-pipeline.yaml

```diff
@@ -50,6 +50,7 @@
                                 sh """
                                 # Disable starting Selenium, Docker is used instead
                                 sed -i 's/"start_process" : true,/"start_process" : false,/g' nightwatch.json
+                                rm -rf ui-testsuite/screenshots ui-testsuite/reports # so that any leftovers from past builds are not archived
```

Comment:
is this required? I thought that the new workspace folder was created for each build. I might be wrong though. And it can't really hurt.

Reply:
Ah, googling a little, it seems that for non-concurrent builds the workspace can be reused, so better to have it like it is.

Reply:
We do need it. Otherwise we might end up archiving stuff from previous builds, see: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-browser-based/4/ (there are three screenshots from three different builds)